sampleconfig/orderer.yaml

```diff
@@ -68,14 +68,14 @@ General:
         # ServerPrivateKey defines the file location of the private key of the TLS certificate.
         ServerPrivateKey:
 
-    # Genesis method: The method by which the genesis block for the orderer
+    # Bootstrap method: The method by which to obtain the bootstrap block
     # system channel is specified. The option can be one of:
     #   "file" - path to a faile containing the genesis block or config block of system channel
     #   "none" - allows an orderer to start without a system channel configuration
-    GenesisMethod: file
+    BootstrapMethod: file
```

Comment:
this PR looks fine by itself but it will break byfn for the same reason before. we need to:
- make this field empty
- change fabric-sample
- re-add value to this field

Reply:
Fabric-sample is updated.

Reply:
fabric-sample is updated.